bullet_train-themes-light/app/views/showcase/previews/partials/_well.html.erb

```diff
@@ -0,0 +1,15 @@
+<% showcase.sample "Basic" do %>
+  <%= render "shared/well" do |well| %>
+    <% well.title "Title" %>
+    <% well.description "I am description" %>
+    <% well.body "I am body" %>
+    <% well.raw_body "<h2>I am body</h2>".html_safe %>
```

Comment:
Let's use the `tag` helper here and call it `raw_body` so it's easier to spot what's what on the page.

```suggestion
    <% well.raw_body tag.h2("I am raw_body") %>
```